the-collab-lab/tcl-25-smart-shopping-list
# Issue #11 Remove items from shopping list

## Description

This PR allows a user to delete items from their shopping list so that their list is not cluttered with items they don't want to buy in the future. To prevent accidental deletions, it asks users to confirm the deletion of the items.The PR ensures that removing items from the app UI removes them from the Firestore db, too.

## Acceptance Criteria


## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Testing Steps / QA Criteria

* From your terminal, pull down this branch with `git pull origin ku-ci-remove-items-from-shopping-list` and check that branch out with `git checkout ku-ci-remove-items-from-shopping-list` 
* Then `npm start` to launch the app
* Check that ``Remove`` button is there
* When clicking on the ``Remove`` button, check that you are asked to either confirm or cancel the deletion
* Make sure the item is deleted both off the list and the Firestore db
* Make sure that the item remains in the list if you cancel the action
